_posts/blog/2019-08-12-why-upgrade.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Top 10 Reasons to Upgrade"
+author: Mike Miller
+reviewers: Keith Turner, Christopher Tubbs
```

Comment:
I don't want my name here. I also don't think any individual names should be here. I think individual names here are too similar to `@author` tags in code, promoting the idea of individuals over community, which is something that the ASF expressly decided against many years ago. The commit logs show the authors and the mailing list show reviewer feedback. That should be sufficient if somebody wants to go back and review that information. For the purpose of the presentation on the website, the content should reflect the community as a whole.

Also, this "reviewers" line is just plain weird to me. It's likely that others reviewed the page, too, but didn't find have any suggestions for changes. Does their passive concurrence count for less?

I don't know... perhaps these should be discussed on the dev@ list, and perhaps the reviewers tag can be addressed separately from the author tag. In either case, I'm not comfortable with my name being added here.

Reply:
I only did those tags because thats what we did here: https://accumulo.apache.org/blog/2016/12/19/running-on-fedora-25.html

Reply:
Feel free to remove the authors and reviewers from posts... doesn't matter to me.

Reply:
I'm still forming my opinions on this one, so I won't take any action until I have a stronger opinion. At the very least, I was surprised to see my name added when I didn't add it. But, I don't blame anybody for following the precedent that was set.